### PR TITLE
Clean up and revisions

### PR DESCRIPTION
**Pull request by trag-stripe**
**Description**
Re-ordering links, resources, and related materials in time for Hacktoberfest 2020

r? @thorsten-stripe 

**Motivation**
We have longstanding updates for this page and additional resources to share with developers.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -35,143 +37,303 @@
...
+                            <li>Create, retrieve, update, and delete <a href="https://stripe.com/docs/cli/resources">API
+                                    objects</a></li>
+                            <li><a href="https://stripe.com/docs/cli/logs/tail">Tail API request logs</a> in real time
+                            </li>
+                            <li><a href="https://stripe.com/docs/cli/listen">Securely test webhooks</a> without
+                                third-party tunneling software</li>
+                        </ul>
                     </p>
-                    <a href="https://github.com/stripe/stripe-payments-demo" class="button" target="_blank">Get the code</a>
+                    <a href="https://stripe.com/docs/stripe-cli" class="button" target="_blank">Launch the
```
Comment (thorsten-stripe) [inline index.html]: Maybe say "Install the CLI" instead?

File: index.html [hunk 2]
```diff
@@ -35,143 +37,303 @@
...
-                        <a href="https://stripe.com/connect" target="_blank">Stripe Connect</a>. The demo application shows how you to quickly onboard users to your platform
-                        and let them collect payments. It uses pre-built UI components to get up and running quickly
-                        while still enabling you to customize the user experience.
-                    </p>
-                    <a href="https://github.com/stripe/stripe-connect-rocketrides" class="button" target="_blank">Get the code</a>
+                        A collection of integrations designed to help you get started with your Stripe integration.
+                        Learn how to:
+
+                        <ul>
+                            <li>Quickly <a href="https://stripe.com/docs/checkout/integration-builder">accept one-time
```
Comment (thorsten-stripe) [inline index.html]: Should this link to https://github.com/stripe-samples/checkout-one-time-payments instead?

File: index.html [hunk 3]
```diff
@@ -35,143 +37,303 @@
...
             </section>
-            <h2>Developer tools</h2>
+            <h2>Developer resources</h2>
             <section class="tools">
-                <a href="https://stripe.com/docs/stripe-cli" target="_blank">
-                    <h3 class="uppercase">Stripe CLI</h3>
-                    <p>Build, test, and manage your integration with Stripe directly from your terminal.</p>
+                <a href="https://stripe.com/docs/development/quickstart" target="_blank">
+                    <h3 class="uppercase">Development quickstart</h3>
+                    <p>Install our client libraries and start developing your Stripe integration.</p>
```
Comment (thorsten-stripe) [inline index.html]: "start building your Stripe integration"?

**Review by thorsten-stripe — APPROVED** [review]
Some minor nits, overall LGTM 👍

**Comment by trag-stripe** [comment]
Applied edits @thorsten-stripe

**Pull request merged**